CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 0.5.2 (September 14, 2016)
+
+* BUG: The ec2-snapper `delete` command now gracefully exits if no snapshots exist for the given instance rather than
+  exiting with an error.
+
```

Comment:
My comment was motivated mainly by how, with Terraform, in particular, I regularly check the CHANGELOG to see what's new. It's nice to go to one file, to see what's in progress before it's published, and to have it all in one place.

But it's a mild preference for me, not a strong one, so if you'd prefer to stick with releases, I'm ok with that.